en/docs/guides/login/oidc-parameters-in-auth-request.md

```diff
@@ -0,0 +1,149 @@
+#Use Advanced Parameters in Authentication Requests
+
+This page guides you through some special request parameters used with Open ID Connect authentication requests.
+
+ ----
+ 
+##[State parameter](../../../concepts/traditional-authentication-request)
+You can use this parameter when you need to maintain a correlation between the request and the response.
+
+```tab="Sample Request"
+https://<host>:/authorize?
+response_type=token
+&client_id=s6BhdRkqt3
+&state=xyz
+&redirect_uri=<callback_url>
+```
+
+```tab="Sample Response"
+HTTP/1.1 302 Found
+Location: <callback_url>#access_token=2YotnFZFEjr1zCsicMWpAA
+&state=xyz
+&token_type=bearer
+&expires_in=3600
+```
+ ----
+ 
+##[Nonce Parameter](../../authentication/concepts/traditional-authentication-request)
```

Comment:
Add a space after ##

also please consider adding the link to the text below rather than linking the heading itself .